### PR TITLE
recipe-client-addon: Add no-shadow lint rule from m-c

### PR DESCRIPTION
**Pull request by mythmon**

### CODE REVIEW

File: recipe-client-addon/lib/CleanupManager.jsm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 "use strict";
 
+const {utils: Cu} = Components;
```
Comment (rehandalal) [inline recipe-client-addon/lib/CleanupManager.jsm]: How did this script ever work without this?

Comment (mythmon) [inline recipe-client-addon/lib/CleanupManager.jsm]: I don't know that it ever did. The changes that needed this are fairly new, and haven't been merged to mozilla-central yet, so it is entirely reasonable that they haven't really been tested.

Comment (rehandalal) [inline recipe-client-addon/lib/CleanupManager.jsm]: Also wondering if there's possibly a better name for `Cu`?

Comment (mythmon) [inline recipe-client-addon/lib/CleanupManager.jsm]: `Cu` is *really* traditional, and used all over the mozilla code base.

Comment (rehandalal) [inline recipe-client-addon/lib/CleanupManager.jsm]: /me nods

**Pull request merged**